README.md

```diff
@@ -12,40 +12,34 @@ running.
 
 1. Clone the `vmctl` repository.
 
-2. Make sure that that `ssh` and `socat` are available. Follow the standard
-   procedure for your distribution to install those packages.
+2. Bootstrap a config directory with the following command:
 
-3. For ease of use, do a symlink in your path, say
+       $ ./vmctl env --confdir $HOME/vms --verbose --bootstrap
 
-       $ ln -s /path/to/vmctl/vmctl $HOME/bin/vmctl
+   **Note** This will:
+    * Ensure that you have `ssh` and `socat` installed
+    * Create a configuration directory in $HOME/vms
+    * Ensure you have a base `*-q35-base.conf`
+    * Ensure you have `common.conf` (to share common vars like
+      `QEMU_SYSTEM_BINARY`)
 
-4. Create a directory to hold your VMs and their configurations
+3. There are two ways of executing `vmctl`:
+    1. Do a symlink in your path to have it available "natively"
 
-       $ mkdir $HOME/vms; cd $HOME/vms
+        $ ln -s /path/to/vmctl/vmctl $HOME/bin/vmctl
 
-5. You probably want to use the `q35-base.conf` configuration file to base your
-   own VMs on, so copy it.
+    2. Always run it with the `env` command:
 
-       $ cp /path/to/vmctl/examples/vm/q35-base.conf .
+        $ ./vmctl env --confdir $HOME/vms
+        $ vmctl --help
 
-6. When you have a lot of configuration there is probably something you'd want
-   to share between them. Examples are `QEMU_SYSTEM_BINARY` and
-   `GUEST_KERNEL_APPEND_EXTRA`. The examples (including the default
-   `q35-base.conf`) assumes the presence of such a `common.conf` file:
-
-       $ cp /path/to/vmctl/examples/vm/common.conf .
-
-7. Start from an example and edit it as you see fit.
-
-       $ cp /path/to/vmctl/examples/vm/nvme.conf .
-
-8. Prepare a boot image. The `q35-base.conf` configuration will look a base
+4. Prepare a boot image. The `*-q35-base.conf` configuration will look a base
    image in `img/base.qcow2`. You can use [archbase][archbase] to build a lean
    Arch Linux base image or grab a QCOW2-based [Ubuntu cloud image][ubuntu-cloud-image]
    if that's your vice.
 
    In the case of a standard "cloud image", you probably want to resize it
-   since it is usually shrinked to be as small as possible by default.
+   since it is usually shrank to be as small as possible by default.
```

Comment:
I think the correct verb conjugation is "shrunk".

Reply:
good catch !!!